src/style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100%;
+  margin: 0;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.image {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.smallimage {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 12px;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir Book, sans-serif;
+}
+
+.search:hover {
+  background: #fff;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
```

Comment:
Have not seen changes from static behavior. Your border value here is the same as at.search